### PR TITLE
Remove no-longer-needed fields from feed spec

### PR DESCRIPTION
**Pull request by mdholloway**
See MCS patch: https://gerrit.wikimedia.org/r/#/c/311733/

Note: Spec-related unit tests will fail if the MCS patch is merged before
this is.

Bug: T146041

### CODE REVIEW

File: v1/feed.yaml [hunk 1]
```diff
@@ -71,18 +71,13 @@ paths:
...
-                    normalizedtitle: /.+/
               random:
                 items:
                   - title: /.+/
               news:
                 - story: /.+/
                   links:
-                    - pageid: /.+/
-                      ns: /.+/
-                      normalizedtitle: /.+/
```
Comment (Pchelolo) [inline v1/feed.yaml]: I thought that `normalizedtitle` was used in the app? No?

Comment (mdholloway) [inline v1/feed.yaml]: Yeah, we do.  Good point, probably better to keep checking for it.  I'm going to do a little more testing and maybe withdraw this PR if we can get back with the spec as-is.

Comment (mdholloway) [inline v1/feed.yaml]: All we really need to do is remove the pageid and ns fields from the expected response for news links.  These aren't used in the app.

**Comment by mdholloway** [comment]
The MCS patch has been merged and deployed.  

I think you'll now see the test "Monitoring routes, en.wikipedia.org domain Retrieve aggregated feed content for April 29, 2016" fail with " AssertionError: Body field pageid not found in response!" until this fix to the spec is made.

**Pull request merged**